docs/installing-and-onfiguring-minikube-environment.md

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
On another not-so-related note, this file too could use an update in the name. I guess it should be "installing-and-**c**onfiguring-minikube-environment.md".